dts/x86/elkhart_lake.dtsi

```diff
@@ -34,16 +34,16 @@
 		#interrupt-cells = <3>;
 	};
 
-	soc {
+	pcie0 {
```

Comment:
The bus nodes are `/soc/pcie0` in elkhart_lake.dtsi, but `/pcie0` in apollo_lake.dtsi. Is that inconsistency deliberate?

Reply:
It's not, I think the pcie0 should be on the top level. I'll fix if.

Reply:
fixed